src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -323,5 +323,14 @@ public void SetComparer_SetEqualsTests()
             Assert.True(comparerSet2.SetEquals(set));
         }
         #endregion
+
+        [Fact]
+        public void IntersectWith_Int ()
```

Comment:
Nit: since there are IntersectWith tests, this name isn't particularly descriptive.  Could you rename it to something like:
```C#
IntersectWith_SupersetEnumerableWithDups_ExpectedResultsInCorrectOrder
```
or something more descriptive like that?